services/dynamic-sidecar/tests/unit/test_models_shared_store.py

```diff
@@ -105,3 +106,24 @@ async def replace_list_in_shared_store(item: str):
         *(replace_list_in_shared_store(f"{x}") for x in range(PARALLEL_CHANGES))
     )
     assert len(shared_store.container_names) == PARALLEL_CHANGES
+
+
+async def test_init_from_disk_with_legacy_data_format(project_tests_dir: Path):
+    MOCKS_DIR = project_tests_dir / "mocks"
```

Comment:
STYLE: why sometimes you use capital and other lowercase for paths? See above and below in this same file